library/pk_wrap.c

```diff
@@ -949,11 +946,12 @@ static int ecdsa_verify_wrap(void *ctx_arg, mbedtls_md_type_t md_alg,
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
     mbedtls_svc_key_id_t key_id = MBEDTLS_SVC_KEY_ID_INIT;
     psa_status_t status;
-    mbedtls_pk_context key;
-    int key_len;
-    unsigned char buf[MBEDTLS_PK_ECP_PUB_DER_MAX_BYTES];
+    size_t key_len;
+    /* This buffer contains first the public key (consisting of two public
+     * points plus a header byte), then the signature (consisting of two
+     * public points). Size it for the public key which is one byte larger. */
```

Comment:
Actually, the public key is one point, encoded as one header byte plus two numbers (in the range [0, p)), while the signature is two numbers (in the range [0, n)). For all curves except secp224k1, p and n have the same byte-size, so indeed a public key is one byte larger. Fortunately, we already [don't support secp224k1 with PSA](https://github.com/Mbed-TLS/mbedtls/issues/3541), and are hoping to [remove it entirely in 4.0](https://github.com/Mbed-TLS/mbedtls/issues/6792#issuecomment-1352905076), so I think it's OK to keep using the pubkey length here.

Please just adjust the comment. Suggesting: "The buffer will contain the public key then the signature at different points in time. For all curves except secp224k1, which is not currently supported in PSA, the public key is one byte longer (header byte + 2 numbers, while the signature is only 2 numbers), so use that as the buffer size."